src/main/groovy/au/org/ala/biocollect/swagger/model/BioCollectModels.groovy

```diff
@@ -309,3 +309,13 @@ class ProjectSearchResponse {
     Integer total
     List<Facet> facets
 }
+
+// classes for "ws/bioactivity/data/simplified/{id}
+@JsonIgnoreProperties('metaClass')
+class GetOutputForActivitySimplifiedResponse {
+    Map activity
+    Map project
```

Comment:
make appropriate changes

Reply:
@temi I've now pushed the code review updates. thank you.